src/components/PageBlocks/CalendarConfigurator/CalendarDisplay.vue

```diff
@@ -24,17 +26,10 @@
         :disabled="options.header.hide"
         :value="true"
```

Comment:
:value="true"
:unchecked-value="false" are legacy things i think. Try without them if it works the same (less code)